Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/FilesStepDefinitions.cs

```diff
@@ -68,5 +86,105 @@ public async Task WhenISendsAGetRequestToTheFilesIdEndpoint(string identityName,
         _responseContext.WhenResponse = await identity.Files.GetFileMetadata(file.Id);
     }
 
+    [When($"{RegexFor.SINGLE_THING} sends a PATCH request to the /Files/{RegexFor.SINGLE_THING}.Id/ClaimFileOwnership with token {RegexFor.SINGLE_THING}.OwnershipToken")]
+    public async Task WhenISendsAPatchRequestToTheFilesFIdClaimFileOwnershipFOwnershipTokenEndpoint(string identityName, string fileName, string fileName2)
+    {
+        var identity = _clientPool.FirstForIdentityName(identityName);
+        var fileId = _filesContext.Files.FirstOrDefault(f => f.Key == fileName).Value?.Id ?? fileName;
```

Comment:
I don't think I like that you're using the file's name as the id. This is quite confusing when reading the tests then. 🤔 
Same goes for the other step definitions